content/blog/2022-07-26-july-22-community-gems.md

```diff
@@ -0,0 +1,230 @@
+tags:
+  - DVC Remotes
+  - Pipelines
+  - CML
+  - DVC Cache
+  - Community
+---
+
+## [How can I track a new file added to my `data` folder if the `data` folder is already tracked by DVC, yet ignored by Git?](https://discord.com/channels/485586884165107732/485596304961962003/983278896587894804)
+
+Great question on how DVC handles data tracking from @NgHoangDat!
+
+Since you already track the `data` folder, when you add a new file into it, all
+you need to do is update your DVC history. You can use either `dvc add data` or
+`dvc commit` to start tracking the new file.
+
+DVC will also only recalculate the changed files. If you add or modify a small
+number of files in that folder, the update will not take very long.
+
+## [What would be the best method to get the remote URL of a given dataset inside a Python environment?](https://discord.com/channels/485586884165107732/485596304961962003/984870485668008007)
+
+Wonderful question from @come_arvis!
+
+You can use the `get_url` method of the
+[DVC Python API](https://dvc.org/doc/api-reference) to do this. Here's an
+example of a script you might run to get the remote URL.
+
+```python
+import dvc.api
+
+resource_url = dvc.api.get_url(
+    'get-started/data.xml',
+    repo='https://github.com/iterative/dataset-registry'
+    )
+
+print(resource_url)
+
+# https://remote.dvc.org/dataset-registry/a3/04afb96060aad90176268345e10355
+```
+
+This URL is built with the project configuration file, `.dvc/config`, and the
+`md5` file hashes stored in the `.dvc` file corresponding to the data file or
+directory you want the storage location of.
+
+## [I'm excited about MLEM helping expose API endpoints to our model, but heard it was experimental. Where can I learn more about how to deploy models with this tool?](https://discord.com/channels/485586884165107732/563406153334128681/992517466662117386)
+
+Great question from @raveman^2!
+
+There are a few ways you can use expose API endpoints to your model:
+
+- Run `mlem serve` to generate a FastAPI endpoint with your model.
+- Export the model as a Python package for your own custom-built API.
+- The experimental deploy to Heroku.
+
+You can find more details here in the MLEM docs: https://mlem.ai/doc/get-started
+
+You can also see an example of deploying a model with MLEM in this
+[blog post tutorial](https://dvc.org/blog/serving-models-with-mlem).
+
+## [How do I revert a `dvc add` command to stop tracking data?](https://discord.com/channels/485586884165107732/563406153334128681/993111134896918599)
+
+This is a good question from @Nwoke!
+
+If you have accidentally added the wrong directory or files for DVC to track,
+you can easily remove them with the `dvc remove` command. This is used to remove
+the `.dvc` file and ensure that the original data file is no longer being
+tracked. Here's an example of this command being used:
+
+```dvc
+$ dvc remove data.csv.dvc
+```
+
+Sometimes when you stop tracking data, you also want to remove it from your
```

Comment:
Should we note that it will remove all data, not only the data that was the target of `dvc remove`?

Reply:
Do you mean for the `dvc gc` command?

Reply:
Yup, sorry for the lack of clarity. I did mean `dvc gc`.